### PR TITLE
Add AXL INU BEP20 and ERC20 Logo and Info

### PR DESCRIPTION
**Pull request by AsserFawzi**
Add AXL INU BEP20 and ERC20 Logo and Info

### CODE REVIEW

**Comment by AsserFawzi** [comment]
Is there's anything I need to do ?

**Comment by Cryptocool1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

File: blockchains/ethereum/assets/0x25b24b3c47918b7962b3e49c4f468367f73cc0e0/info.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AXL INU",
+    "website": "https://www.axltoken.com/",
+    "description": "The Ultimate Decentralized Platform",
+    "explorer": "https://etherscan.io/token/0x25b24b3c47918b7962b3e49c4f468367f73cc0e0",
+    "type": "ERC20",
+    "symbol": "AXL",
+    "decimals": 18,
+    "status": "active",
+    "id": "0x25b24b3c47918b7962b3e49c4f468367f73cc0e0",
```
Comment (cryptomanz) [inline blockchains/ethereum/assets/0x25b24b3c47918b7962b3e49c4f468367f73cc0e0/info.json:10]: hello,
id and folder should be in checksum format, please change

**Comment by cryptomanz** [comment]
folder info.json and logo.png should be in checksum format, please change

**Comment by Cryptocool1** [comment]
Please create new PR, follow doc https://developer.trustwallet.com/assets/requirements
You may want to try our Assets app in beta; go to https://assets.trustwallet.com, log on with Github, enter contract ID, logo and some additional info, and it will create a PR for you.
See https://developer.trustwallet.com/assets/repository_details

**Comment by Cryptocool1** [comment]
make sure to mention this PR so the fee can be reused

**Pull request closed without merging**